From f29f35fcada9a0c3df0b1cc57ffcc7c8ba3c1b2a Mon Sep 17 00:00:00 2001
From: Andreas Kempe <kempe@lysator.liu.se>
Date: Tue, 8 Jan 2019 20:47:50 +0100
Subject: [PATCH] config: Use one equal sign for tests in configure.ac

The test utility expects one equal sign when comparing strings as per
the man page. Correct string equality tests to use a single equal sign
to avoid errors during the configuration process.

Upstream-Status: Submitted
Patch-URI: https://github.com/tieto/sipe/pull/199
---
 configure.ac | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index ca21239e2..e7881b084 100644
--- a/configure.ac
+++ b/configure.ac
@@ -423,10 +423,10 @@ AS_IF([test "x$enable_purple" != xno],
 					[PKG_CHECK_MODULES(_SIPE_MEDIA_RECHECK_PLEASE_IGNORE, [$purple_pkgconfig >= 2.8.0],
 						dnl check purple pkgconfig for gstreamer version
 						[gstreamer_pkgconfig=`$PKG_CONFIG --variable=gstreamer $purple_pkgconfig`
-						AS_IF([test "x$gstreamer_pkgconfig" == x],
-							[AS_IF([test "x$purple_pkgconfig" == xpurple-3],
+						AS_IF([test "x$gstreamer_pkgconfig" = x],
+							[AS_IF([test "x$purple_pkgconfig" = xpurple-3],
 								[gstreamer_pkgconfig=1.0])])
-						AS_IF([test "x$gstreamer_pkgconfig" == x1.0],
+						AS_IF([test "x$gstreamer_pkgconfig" = x1.0],
 							[gstreamer_pkgconfig="gstreamer-$gstreamer_pkgconfig gstreamer-rtp-$gstreamer_pkgconfig"
 
 							AC_MSG_NOTICE([using packages "$gstreamer_pkgconfig".])
@@ -535,18 +535,18 @@ AS_IF([test "x$with_purple_vv" != xno],
 AM_CONDITIONAL(SIPE_WITH_VV, [test "x$with_purple_vv" != xno])
 
 dnl raw data RTP streams enable Lync file transfer
-AM_CONDITIONAL(SIPE_HAVE_XDATA, [test "x$ac_have_xdata" == xyes])
-AS_IF([test "x$ac_have_xdata" == xyes],
+AM_CONDITIONAL(SIPE_HAVE_XDATA, [test "x$ac_have_xdata" = xyes])
+AS_IF([test "x$ac_have_xdata" = xyes],
 	[AC_DEFINE(HAVE_XDATA, 1, [Define if we have raw data RTP in media backend.])])
 
 dnl enable appshare support
-AM_CONDITIONAL(SIPE_HAVE_APPSHARE, [test "x$ac_have_appshare" == xyes])
-AS_IF([test "x$ac_have_appshare" == xyes],
+AM_CONDITIONAL(SIPE_HAVE_APPSHARE, [test "x$ac_have_appshare" = xyes])
+AS_IF([test "x$ac_have_appshare" = xyes],
 	[AC_DEFINE(HAVE_APPSHARE, 1, [Define to 1 if we have appshare support.])])
 
 dnl RDP server for sharing local desktop
 AM_CONDITIONAL(SIPE_HAVE_APPSHARE_SERVER, [test "x$ac_have_appshare_server" = xyes])
-AS_IF([test "x$ac_have_appshare_server" == xyes],
+AS_IF([test "x$ac_have_appshare_server" = xyes],
 	[AC_DEFINE(HAVE_APPSHARE_SERVER, 1, [Define if appshare server is enabled.])])
 
 dnl telepathy code parts rely on interfaces that require GValueArray. This
